standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java

```diff
@@ -1315,4 +1316,35 @@ public static String getHttpPath(String httpPath) {
     }
     return httpPath;
   }
+
+  public static String convertSqlPatternToRegExp(String likePattern) {
```

Comment:
it looks like a copy&paste from `UDFLike.likePatternToRegExp`

Reply:
ok, why not add param `bool quote` and reuse the method

Reply:
Addressed, need make some minor change on this method.